### PR TITLE
Update docs 2.0

### PR DESCRIPTION
**Pull request by stfung77**
### What
Update config documentation and the "Running & Configuring the Application" document to reflect the new config schema changes and removal of Circle support

### Why
Old documentation is no longer accurate

### CODE REVIEW

File: docs/01 - Running & Configuring the Application/A - Running & Configuring the Application.md [hunk 1]
```diff
@@ -18,31 +19,42 @@
...
 ## Configuring the Project
 
 ### Config Files
 
-As mentioned previously, both the Anchor Platform and Anchor Reference server are configured using yaml and they have default configuration files:
+As mentioned previously, both the Anchor Platform and Anchor Reference server are configured using yaml and they have 
+default configuration files:
 
-- **Anchor Platform** default config file is located at [`anchor-config-defaults.yaml`].
+- **Anchor Platform** default config file is located at [`anchor-config-defaults-values.yaml`].
```
Comment (JakeUrban) [inline docs/01 - Running & Configuring the Application/A - Running & Configuring the Application.md:47]: Is it `anchor-config-defaults-values.yaml` or `anchor-config-default-values.yaml`?

**Review by lijamie98 — APPROVED** [review]
LGTM

File: docs/01 - Running & Configuring the Application/A - Running & Configuring the Application.md [hunk 2]
```diff
@@ -130,64 +196,70 @@ Note: secrets (credentials, tokens, etc...) are passed to the application via en
...
+  extras_enabled: true     # optional metrics that periodically query the database
+  run_interval: 30         # interval (seconds) to query the database for the extra metrics
 ```
 
 A Grafana dashboard for the Anchor Platform can be found at `docs/resources/grafana-dashboard/anchor-platform-grafana-dashboard.json`
 and imported into your Grafana instance to visualized the Prometheus metrics.
 
 
-[`anchor-config-defaults.yaml`]: ../../platform/src/main/resources/anchor-config-defaults.yaml
+[`anchor-config-defaults.yaml`]: ../../platform/src/main/resources/config/anchor-config-default-values.yaml
```
Comment (JakeUrban) [inline docs/01 - Running & Configuring the Application/A - Running & Configuring the Application.md:259]:
```suggestion
[`anchor-config-default-values.yaml`]: ../../platform/src/main/resources/config/anchor-config-default-values.yaml
```

File: docs/01 - Running & Configuring the Application/A - Running & Configuring the Application.md [hunk 3]
```diff
@@ -81,27 +97,76 @@ The Platform configuration loader tries to fetch the configuration from up to th
...
+# REQUIRED - JWT secrets used to communicate between Anchor and Platform.
+SECRET_CALLBACK_API_AUTH_SECRET=<secret>
+SECRET_PLATFORM_API_AUTH_SECRET=<secret>
+```
 
 ### Supported Assets
 
-The Anchor Platform reads the list of supported assets from a json file whose address is configured in the config file under `app-config.app-assets` and defaults to [`assets-test.json`] ([ref](https://github.com/stellar/java-stellar-anchor-sdk/blob/1f84429f0c5d35cee75445686242643fbd8cffa5/platform/src/main/resources/anchor-config-defaults.yaml#L74)).
+The Anchor Platform reads the list of supported assets (to be set in the configuration `override` file) in json form, 
+an example can be found in ['example.anchor-config.yaml'] under `assets`.
```
Comment (JakeUrban) [inline docs/01 - Running & Configuring the Application/A - Running & Configuring the Application.md:161]: `['example.anchor-config.yaml'] isn't matching any link added to the bottom of the file

**Review by JakeUrban — APPROVED** [review]

**Pull request merged**